packages/fiori3/src/components/AnalyticalTable/demo/demo.stories.tsx

```diff
@@ -0,0 +1,126 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { boolean, number } from '@storybook/addon-knobs';
+import { AnalyticalTable as Table } from '../index';
```

Comment:
import from `lib`, and in case nothing is conflicting please use the `Analytical` import